pkg/plugins/hybrid/v1alpha/scaffolds/internal/templates/makefile.go

```diff
@@ -109,6 +109,23 @@ all: build
 help: ## Display this help.
 	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
 
+##@ Build
+.PHONY: build
+build: generate fmt vet ## Build manager binary.
+	go build -o bin/manager main.go
+
+.PHONY: run
+run: helm-operator ## Run against the configured Kubernetes cluster in ~/.kube/config
+	go run ./main.go
+
+.PHONY: docker-build
```

Comment:
@skattoju Looks like `docker-build` and `docker-push` targets are already there in the Makefile (https://github.com/operator-framework/helm-operator-plugins/blob/6e30bde2868879811b3bedb200e911ae440413c8/testdata/hybrid/memcached-operator/Makefile#L69-L73)

Reply:
We can remove one of them.

Reply:
whoops botched merge 😅 , fixed 👍